lib/lgy/service.rb

```diff
@@ -139,7 +121,7 @@ def get_coe_documents
       end
     end
 
-    def get_document(id:)
```

Comment:
It doesn't matter to me either way, but just curious if there was a reason to switch from named params to positional params here for this?

Reply:
Yeah I figured since there is only 1 parameter it was fine to change this one